src/App.js

```diff
@@ -1,22 +1,37 @@
 import "./App.css";
 import React, { useRef, useEffect, useState } from "react";
+import Navbar from './components/Navbar';
 // eslint-disable-next-line import/no-webpack-loader-syntax
-import Map from "./components/Map.js";
-import SurfSpotCard from "./components/SurfSpotCard";
+//import Dashboard from "./components/Dashboard.js";
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import Location2 from './pages/Location2';
+import Location1 from './pages/Location1';
+import Home from './pages/Home.js';
+//import LoginForm from "./components/LoginForm";
 
 <link
   rel="stylesheet"
   href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap"
 />;
 
+const testUser = {
+  email: "carlo@dumb.com",
+  password: "imstupid",
+};
 
 function App() {
   return (
-    <div>
-      <Map />
-      
-    </div>
+    <>
+      <Router>
+        <Navbar />
```

Comment:
New routing is clean. It makes sense. Just need to merge my pages with yours and we will be in tip top shape.